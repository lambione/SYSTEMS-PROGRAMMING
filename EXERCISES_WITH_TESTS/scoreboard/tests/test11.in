3 2 10 2
2 1 1 0
1 3 100 101
1 4 0 0
1 5 0 0
1 6 0 0
1 7 0 0
1 8 0 0
1 9 0 0
1 10 0 0
10 4 0 0
10 5 0 0
10 6 0 0
10 7 0 0
10 8 0 0
10 9 0 0
10 1 0 0
