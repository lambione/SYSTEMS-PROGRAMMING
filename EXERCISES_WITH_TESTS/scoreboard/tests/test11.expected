1 8 10
10 8 8
3 6 2
2 3 2
4 2 2
5 2 2
6 2 2
7 2 2
8 2 2
9 2 2
